### ISSUE TITLE
Too many false positives on test-data using test.py

### ISSUE THREAD

**Issue by davidkvcs** [Project-MONAI/research-contributions#93]
**Describe the bug**
I get too many false positives in areas that should be air when running test.hnc.

**To Reproduce**
Train Swin-UNETR as in this repo, but using a dataset of HNC patients (where relevant volume is much smaller then with segmentation of the abdomen).

Then run on test-set and plot the segmentation results.

**Expected behavior**
Expect higher quality segmentation. 

**Screenshots**
![image](https://user-images.githubusercontent.com/37078212/186920603-a5998d4b-785d-4b01-8137-183f9560f441.png)
The turqoise area is where ground truth and prediction overlaps.
The purely false positive areas are shown in green outside the body. 

**Environment (please complete the following information):**
Same as in issue: https://github.com/Project-MONAI/research-contributions/issues/67

 - GPU models and configuration
Same as in issue: https://github.com/Project-MONAI/research-contributions/issues/67

**Additional context**
I suspect this could have to do with [data_utils.py](https://github.com/Project-MONAI/research-contributions/blob/main/SwinUNETR/BTCV/utils/data_utils.py). 

I noticed that the test_transform of the dataloader does not have the line `transforms.CropForegroundd(keys=["image", "label"], source_key="image")`.
I suspect this might be a larger problem in my use case, since there is a lot more air in the image. 
However, if I add the `transforms.CropForegroundd(keys=["image", "label"], source_key="image")` to the test_transform in line 152, this yields a cropped prediction result with the wrong dimensions. 

If my hypothesis is correct, can you please let me know, how I can do cropping, so the model only sees the cropped data (as during training), but then somehow padding the segmentation result, so I get the predicted labels in the correct original patient space? 

Thank you.
